kronos-java/src/main/java/com/lyft/kronos/internal/ntp/SntpClient.java

```diff
@@ -184,30 +198,6 @@ static long readTimeStamp(byte[] buffer, int offset) {
         return ((seconds - OFFSET_1900_TO_1970) * 1000) + ((fraction * 1000L) / 0x100000000L);
     }
 
-    /**
-     * Writes system time (milliseconds since January 1, 1970) as an NTP time stamp
-     * at the given offsetMs in the buffer.
-     */
-    private static void writeTimeStamp(byte[] buffer, int offset, long time) {
```

Comment:
`readTimeStamp` can be removed as well but unfortunately it’s used in tests. I’m not sure it was a good idea to use the code under test in tests but here we are. This will change in future PRs on improving the test coverage for this class.